auth-api/src/main/java/io/stargate/auth/api/impl/AuthApiServerFactory.java

```diff
@@ -33,6 +33,9 @@
 @JsonTypeName("auth-api")
 public class AuthApiServerFactory extends SimpleServerFactory {
 
+  private static final boolean REQUEST_LOGS_ENABLED =
+      Boolean.parseBoolean(System.getProperty("stargate.requests_logging.enabled", "true"));
```

Comment:
Wouldn't it be better to go with some drop-wizard built-in properties, I am not sure if this is possible, but we could look into it: https://www.dropwizard.io/en/latest/manual/configuration.html#logging

Reply:
Please look at the ticket description for more details. I've tried all approaches.

Reply:
@tomekl007 Have a look here for a solution: https://stackoverflow.com/questions/45484804/pass-an-empty-array-to-dropwizard-via-command-line

`-Ddw.server.requestLog.appenders=,`

Reply:
Unfortunately, the same problem for both:
```
Caused by: com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot deserialize instance of `java.util.ArrayList<io.dropwizard.logging.AppenderFactory<ch.qos.logback.access.spi.IAccessEvent>>` out of VALUE_STRING token
```